src/main/scala/scalalab3/chatbotengine/telegram/TelegramLongPoolingEngine.scala

```diff
@@ -0,0 +1,16 @@
+package scalalab3.chatbotengine.telegram
+
+import akka.actor.ActorSystem
+
+import scala.collection.mutable.ListBuffer
+import scalalab3.chatbotengine.core._
+
+class TelegramLongPoolingEngine extends ChatBotEngine {
+  var registeredBots: scala.collection.mutable.ListBuffer[ChatBot] = ListBuffer.empty
```

Comment:
Let's avoid 'var' for any case. Use Builder pattern here. 
I could write example if you need it :-)